controller/coq_lsp.ml

```diff
@@ -544,9 +541,17 @@ let lsp_main bt coqlib vo_load_path ml_include_path =
 
   (* Input/output will happen now *)
   try
+    (* Core Coq initialization *)
+    let cmdline =
+      { Coq.Workspace.CmdLine.coqlib; vo_load_path; ml_include_path }
+    in
+
     (* LSP Server server initialization *)
     let workspace = lsp_init_loop ic oc ~cmdline in
 
+    let fb_queue = Coq.Protect.fb_queue in
+    let root_state = coq_init ~fb_queue ~bt in
```

Comment:
The order here has to go back to `coq_init` first, then workspace init after.

That's not a big deal workspace initialization can take care of the debug stuff.

But indeed backtraces are special, for example you often want them to be enabled before the lsp_init loop takes place.

Reply:
If I don't do this then backtraces are not picked up from the settings. What is your suggestion for this?

Reply:
Have `Workspace.apply` set it too, you'll need to add the option to the particular workspace setting.